emission/tests/storageTests/TestTimeSeries.py

```diff
@@ -81,6 +81,30 @@ def testExtraQueries(self):
         with self.assertRaises(AttributeError):
             list(ts.find_entries(time_query=tq, extra_query_list=[ignored_phones]))
 
+    def testFindEntriesCount(self):
+        '''
+        Test: Specific keys with other parameters not passed values.
+        Input: For each dataset: ["background/location", "background/filtered_location]
+            - Testing this with sample dataset: "shankari_2015-aug-21", "shankari_2015-aug-27"
+        Output: Aug_21: [738, 508], Aug_27: [555, 327]
+            - Actual output just returns a single number for count of entries.
+            - Validated using grep count of occurrences for keys: 1) "background/location"     2) "background/filtered_location"
+                - $ grep -c <key> <dataset>.json
+        '''
+        ts1_aug_21 = esta.TimeSeries.get_time_series(self.testUUID1)
+        ts2_aug_27 = esta.TimeSeries.get_time_series(self.testUUID)
+
+        count_ts1 = [ts1_aug_21.find_entries_count(key="background/location"), ts1_aug_21.find_entries_count(key="background/filtered_location")]
+        print("\nEntry counts for location, filtered_location on {} = {}".format("Aug_21", count_ts1))
+        self.assertEqual(count_ts1, [738, 508])
+
```

Comment:
These are very basic tests - I don't see any corner-case testing, for example.
- What happens if the key list is blank?
- What happens if the key list is `blablabla`?
- What happens if there is no data for that user?

I believe you had listed out a few similar cases during our discussion, I don't see any actual implementation of those here.

Reply:
I had not implemented for blank keys as it was to be decided whether key is to be made mandatory or not and assumed that a key would always be passed for getting count. 
But yes, I will go ahead and make code changes to accommodate for blank keys by returning total counts from each of the two timeseries DBs as a tuple (count_timeseries_db, count_analysis_db).

Reply:
I had a doubt, regarding `"key list"`. For now, I have only been taking a single key as opposed to a list.
1) Is the count to be done for multiple keys passed as a key list?
2) If yes, ´get_query'  returns a combined query for multiple keys and gives total count of matching entries not segregated by keys. For instance, keylist = [A,B], then total count will be (A+B).
Do we need a separate count for each key in that case, that is (A,B) or combined count = (A+B) is needed?

Reply:
Please see https://github.com/e-mission/e-mission-server/pull/935#issuecomment-1707079925